### PR TITLE
Enable edge rate limiting

### PR DESCRIPTION
**Pull request by theseanything**
The allows us to rate limit requests at edge.

### CODE REVIEW

File: modules/assets/main.tf [hunk 1]
```diff
@@ -94,6 +94,24 @@ resource "fastly_service_vcl" "service" {
     }
   }
 
+  rate_limiter {
+    name = "rate_limiter"
+
+    rps_limit            = 500
```
Comment (sengi) [inline modules/assets/main.tf:100]: Makes sense — presumably the idea is to enable it first like this as an expected no-op for a couple days, confirm no weird surprises and then reduce the limit to something meaningful like say 50 rps.

**Review by sengi — APPROVED** [review]
Yay!

**Comment by sengi** [comment]
We also have the option (later, once we're happy that the basics are working as intended) to set stricter limits from the sketchier parts of the ripe/apnic v4/v6 space to filter out some of the less sophisticated/exploratory crud that we currently just absorb.

File: modules/assets/main.tf (hunk 1 above)
Comment (theseanything) [reply modules/assets/main.tf:100]: Yup. I know things like the mirror scrape script are probably higher than that 150req/s - so will need to think about the threshold or slow that stuff down.

**Pull request merged**